### PR TITLE
AFKR-12: Providing a minimal set of Bahmni reports

### PR DESCRIPTION
**Pull request by rubailly**
- I enabled the reports app from the home app dashboard

- The reports app was not provided by easydoc's bahmni_config, nor the parent project. I had to download it from default bahmni_config on GitHub

- `CHOOSE_REPORT_FORMAT` i18n for French locale was missing. Unfortunately, this was also not available in default bahmni_config, the French locale file is provided bay bahmniapps.
I checked out the whole file to override the one from bahmniapps and provided it in easydoc's bahmni_config

- I removed most of the reports and left only 5 reports as requested on the tickets. The reports I left don't use any SQL from global properties, they are default reports that come with Bahmni.

### CODE REVIEW

File: configuration/openmrs/i18n/reports/locale_fr.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "userName": "Nom d'utilisateur",
+    "password": "Mot de passe",
+    "location": "Emplacement",
+    "login": "S'identifier",
+    "selectLocation": "Choisir emplacement",
+    "clinical_key": "Clinique",
```
Comment (rbuisson) [inline configuration/openmrs/i18n/reports/locale_fr.json]: Do you know what those are for?
Maybe we could just remove them.

File: configuration/openmrs/apps/reports/sql/testCount.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+SELECT DISTINCT
```
Comment (rbuisson) [inline configuration/openmrs/apps/reports/sql/testCount.sql]: Does this file have any use?

File: configuration/openmrs/apps/reports/reports.json [hunk 3]
```diff
@@ -0,0 +1,88 @@
+{
+  "visitReport10": {
+    "name": "Visit Report",
```
Comment (rbuisson) [inline configuration/openmrs/apps/reports/reports.json:3]: This suggests the name may not be i18n-able. Do you confirm?

**Review by rbuisson — CHANGES_REQUESTED** [review]
Looks quite good already. Thanks.
I have left few comments.
Also, for the translation JSON file, I have reviewed it. Please replace by this one: https://paste.ubuntu.com/p/ZrYBNNRmNg/
 👍

File: configuration/openmrs/apps/reports/sql/testCount.sql (hunk 2 above)
Comment (rubailly) [reply configuration/openmrs/apps/reports/sql/testCount.sql]: Not for easyDoc (for now). It's a default bahmni file and is used for [a lab report ](https://github.com/Bahmni/default-config/blob/master/openmrs/apps/reports/reports.json#L167) for counting lab tests. 
[I've removed the file.](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/14/commits/6a6330730a27226c8a8ebcc0289d129b2be6e2f9)

File: configuration/openmrs/apps/reports/reports.json (hunk 3 above)
Comment (rubailly) [reply configuration/openmrs/apps/reports/reports.json:3]: Yes, report names are not translatable at the moment. There is a missing` | translate` in [bahmniapps](https://github.com/Bahmni/openmrs-module-bahmniapps/blob/master/ui/app/reports/views/reports.html#L36).
I verified translating the report names by adding it to bahmniapps in my vagrant and it worked. Maybe this should be added into bahmniapps master.

File: configuration/openmrs/i18n/reports/locale_fr.json (hunk 1 above)
Comment (rubailly) [reply configuration/openmrs/i18n/reports/locale_fr.json]: I couldn't find where there are used within the reports app. -I removed those.

File: configuration/openmrs/apps/reports/reports.json (hunk 3 above)
Comment (rbuisson) [reply configuration/openmrs/apps/reports/reports.json:3]: OK. That would be great yes.
Could you create the PR to Bahmni Apps?

Comment (rubailly) [reply configuration/openmrs/apps/reports/reports.json:3]: https://github.com/Bahmni/openmrs-module-bahmniapps/pull/295

Comment (rbuisson) [reply configuration/openmrs/apps/reports/reports.json:3]: Thanks, I'll merge that Bahmni Apps PR.
In the mean time, let's add the Reports as is.

**Pull request merged**